leobot_web_server/web_content/index.html

```diff
@@ -20,6 +21,14 @@ <h1>OPEN SOURCE TELEPRESENCE ROBOT</h1>
             <section class="video">
                 <h3 class="video-header">VIDEOSTREAMING FROM CAMERA 1</h3>
                 <iframe class="video-streaming" src="http://0.0.0.0:8090/stream?topic=/leobot/stereocamera/left/image_raw&width=640&height=470" frameborder="0" gesture="media" allow="encrypted-media" allowfullscreen></iframe>
```

Comment:
It should work on remote machine as well. Please build URL of machine not assume that it is on local machine